app/src/main/kotlin/com/wire/android/feature/auth/client/datasource/remote/ClientRemoteDataSource.kt

```diff
@@ -35,4 +39,13 @@ class ClientRemoteDataSource(
     ): Either<Failure, Unit> = request {
         clientApi.updatePreKeys(authorizationToken, clientId, updatePreKeysRequest)
     }
+
+    suspend fun clientIdsOfUsers(
+        authorizationToken: String,
+        userIds: List<QualifiedId>
+    ): Either<Failure, Map<QualifiedId, List<String>>> = request {
+        val dtos = userIds.map(clientRemoteMapper::fromQualifiedIdToQualifiedIdDTO)
```

Comment:
What the meaning of `dtos` here?

Reply:
Data Transfer Object.

Used to represent a data structure that is commonly known between different applications. Like "Api Model".